yargs.js

```diff
@@ -173,41 +174,49 @@ function Yargs (processArgs, cwd, parentRequire) {
   }
 
   self.boolean = function (keys) {
+    argsert('<array|string>', [].slice.call(arguments))
```

Comment:
Why the `[].slice.call(arguments)`? This seems horrifically ugly. =D (Uhm, in the nicest way possible.)

Is `argsert` trying to `forEach` the args part? Why not change that to a C-style for loop and save yourself this weirdness?